raiden-dapp/src/views/SelectToken.vue

```diff
@@ -57,15 +57,14 @@
 
 <script lang="ts">
 import { Component, Mixins } from 'vue-property-decorator';
-import AddressInput from '@/components/AddressInput.vue';
 import { mapGetters } from 'vuex';
 import { Token } from '@/model/types';
 import NavigationMixin from '@/mixins/navigation-mixin';
 import BlockieMixin from '@/mixins/blockie-mixin';
 import ListHeader from '@/components/ListHeader.vue';
 
 @Component({
-  components: { ListHeader, AddressInput },
+  components: { ListHeader },
```

Comment:
Randomly found this and seems to be unused, please verify

Reply:
Yes, it should be unused. Thanks for spotting.